conformance/conformance.proto

```diff
@@ -57,11 +57,13 @@ option java_package = "com.google.protobuf.conformance";
 //   2. parse the protobuf or JSON payload in "payload" (which may fail)
 //   3. if the parse succeeded, serialize the message in the requested format.
 message ConformanceRequest {
+  string test_name = 1;
```

Comment:
Why don't we change the runner to log the request as well as the test name? This is rather an implementation detail of the runner. If the testee doesn't use it, I don't think it should be in the protocol.